stats/stats/src/chart/mock.rs

```diff
@@ -1,3 +1,4 @@
+use crate::counters_list;
```

Comment:
do we need `chart/mock.rs`? It seems that those functions are not used anywhere

Reply:
For now, we didn't. But this function will be useful for mocking lines chart